packages/kit/test/utils.js

```diff
@@ -110,6 +110,90 @@ export const test = base.extend({
 		}
 
 		use(read_errors);
+	},
+
+	start_server: async ({}, use) => {
+		/**
+		 * @type {http.Server}
+		 */
+		let server;
+
+		/**
+		 * @type {Set<import('net').Socket>}
+		 */
+		let sockets;
```

Comment:
it seemed to be working without all the socket craziness before. do we really need to add the extra complication?

Reply:
this is what i found as advice for force closing http.Server.  I had logging locally and it actually does call socket.destroy in some cases, which should speed up the close at best and prevent timeouts at worst.

Given that this lives in a fixture and is out of the way for tests, not even used unless you actively call start_server(), i'd say it's worth it.